app/src/main/java/com/appharbr/example/app/max/MaxBannerRecyclerViewActivity.java

```diff
@@ -0,0 +1,143 @@
+package com.appharbr.example.app.max;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+import android.widget.FrameLayout;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.appharbr.example.app.databinding.ActivityMaxRecyclerViewBinding;
+import com.appharbr.example.app.databinding.RecyclerViewItemBannerBinding;
+import com.appharbr.sdk.engine.AdSdk;
+import com.appharbr.sdk.engine.AppHarbr;
+import com.appharbr.sdk.engine.listeners.AHListener;
+import com.applovin.mediation.MaxAd;
+import com.applovin.mediation.MaxAdFormat;
+import com.applovin.mediation.MaxAdViewAdListener;
+import com.applovin.mediation.MaxError;
+import com.applovin.mediation.ads.MaxAdView;
+import com.applovin.sdk.AppLovinSdk;
+import com.applovin.sdk.AppLovinSdkUtils;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Arrays;
+
+public class MaxBannerRecyclerViewActivity extends AppCompatActivity {
+
+    ActivityMaxRecyclerViewBinding binding;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+	super.onCreate(savedInstanceState);
+
+	binding = ActivityMaxRecyclerViewBinding.inflate(getLayoutInflater());
+	setContentView(binding.getRoot());
+
+	//Initialize AppLovinSdk
```

Comment:
The comments suppose to be like a tutorial with steps.

Reply:
`//      **** (1) **** 
 //      Max mediation initialization`

Like this?

Reply:
I think we do not need to add the Initialization as the first step.
They should do it anyway regardless of the ad format.

Reply:
aa you mean we do not need to mark comments with stars and numbers? 
So to say we do not need to include this as step?